cli/commands/project_sync.go

```diff
@@ -0,0 +1,235 @@
+package commands
```

Comment:
There is really no concept of sub packages. They are all "Standalone" I did this previously, but since there is a BaseCommand we extend, I then got an error with cyclical imports. It was a mess, but no way around it. Since there are not subpackages it seems like it would just be easier if project_config.go was just moved into the main commands package.